### PR TITLE
Adding Time::remainingTime()

### PR DESCRIPTION
**Pull request by burzum**
Adding a method to get the remaining time from a given datetime.

I'm not sure about the output / return value, any suggestions are welcome. I also considered adding a 2nd, optional arg that can be the date the first arg is compared to.

### CODE REVIEW

**Comment by ionas** [comment]
What about ```Time->difference($timeA, $timeB);``` which could also return negatives and ```$timeA``` and ```$timeB``` could also take ```'now'``` as a static string for comparison.

For convenience If the 2nd parameter is not set it is assumed to be ```'now'```.

```Time::difference($inFuture);``` would equal the suggestion here, then, that is ```Time::remainingTime($inFuture);```.

**Comment by ravage84** [comment]
:+1:  ``remaintingTime()`` sounds good, but don't feature bloat it with adding a parameter that extends the function beyond it's scope and what its name says.

**Comment by ionas** [comment]
```remainingTime()``` should never return a negative value though. IMHO. That's the reason for the suggestion above.

**Comment by dereuromark** [comment]
You could also throw an exception if you insert incomparable dates.

**Comment by ravage84** [comment]
:+1: for an exception in that case.
Makes sense, easily catchable and easy to document.

File: src/I18n/Time.php [hunk 1]
```diff
@@ -799,6 +799,18 @@ public static function parseTime($time, $format = null)
     }
 
     /**
+     * Convenience method for getting the remaining time from a given time.
+     *
+     * @param DateTime $datetime The date to get the remaining time from.
+     * @return DateInterval|boolean The DateInterval object representing the difference between the two dates or FALSE on failure.
+     */
+    public static function remainingTime($datetime)
```
Comment (markstory) [inline src/I18n/Time.php]: `fromNow()` might make it clearer what this method does and remove the need to handle the cases @ravage84 and @ionas raised without needing an exception.

**Comment by jhli** [comment]
I am also :-1: on exceptions cause it might be legit to have negative values that should ve returned simly as zero difference. To me though remainingTime makes me expect 0 or greater whereas for fromNow I'd not be sure what to expect (negatives yes/no?)

**Comment by markstory** [comment]
@burzum Ping.

**Comment by burzum** [comment]
@markstory discussion is over? So what do you want now? Just rename it?

**Comment by markstory** [comment]
@burzum Yes, I think a rename removes the need to handle solutions and lets the method handle values in the past and future.

**Comment by burzum** [comment]
@markstory done :)

**Pull request merged**